spec-0004/index.md

```diff
@@ -91,26 +91,36 @@ Complete examples of how projects implement this in their CI setup are linked in
 #### Process for Adding New Projects
 
 After someone creates an issue on https://github.com/scientific-python/upload-nightly-action
-requesting access to upload wheels a human has to respond to that request.
+requesting access to upload wheels a human/admin has to respond to that request.
+
+Admins are people from the community who are ideally not part of the same organizations nor
+project. This is to prevent malicious activities from a given group of actors and ensure a
+diverse and healthy community. Adding new people to the list requires of admins requires at
+least an issue to be openned.
 
 We want to be open to projects uploading wheels but at the same time need to perform some
 amount of due dilligence before giving people access. This is because once a user is given
-access they could upload wheels for any project. We assume that people are not malicious
-and we can see from the logs who misbehaved after the fact.
+access they benefit the wide exposure of Scientific Python and could take this to their
+advantage to publish malicious packages.
 
 Once you have established who the person is and that they represent the project they want
 to upload wheels for ask the person to create an account on https://anaconda.org and tell
-you the username.
-
-You can then add them to the `scientific-python-nightly-wheels` organisation on anaconda.org.
-Let the user know that they have been added and that they can create a access token at
-https://anaconda.org/scientific-python-nightly-wheels/settings/access. The token should
-only have the "Allow uploads to Standard Python repositories" and
-"Allow write access to the API site" scope. It should use the project name as the token name.
-
-You can also add new people to https://anaconda.org/scientific-python-nightly-wheels when they
-contact you privately, but it would be good if an issue is created as part of that. This helps
-with keeping track of who did what when.
+you the username. We suggest projects to have a least 2 representatives and ideally they
+should not be part of the same organization to ensure a diverse community.
+
+You (as an admin) can then generate a personal access token at
+https://anaconda.org/[user]/settings/access.
+The token should only have the "Allow uploads to Standard Python repositories" and
+"Allow write access to the API site" scope. The creation of tokens at the organization
+level should be avoided for security reasons.
+
+Then you need to do a first upload of a wheel to create the package listing on anaconda.org.
```

Comment:
Do you mean that an existing admin uploads the wheel for the new admin? Or is the "you" here switching from the previous paragraph?

Reply:
Yeah this part is annoying. Here I was suggesting that an admin of scientific-python would do the first upload, otherwise you can communicate the token to the new person. Which is an issue as you would communicate an admin token to someone else and this person could either do a rogue action or if not it's just an admin token in the nature and you really must not forget to revoke the token once the package is up.

Reply:
Ahhh ok! If the admin is still referring to the same person then I have no issue with this phrasing.